README.md

```diff
@@ -41,15 +41,25 @@ Here is a list of projects that you could contribute to during the spree:
 
 | Project      			                                                                          | Contact             |
 | ----------------------------------------------------------------------------------------- | ------------------- |
-| Orkestra (https://github.com/Orkestra-Tech/orkestra)                | @joan38           |
-| [Chimney](https://github.com/scalalandio/chimney/) - Scala library for boilerplate-free data transformations | @krzemin |
-| [julienrf/endpoints][endpoints] - API definition shared between client, server and docs ([issues][endpoints-issues])   | [@Krever][krever]   |
-| [Bloop](https://github.com/scalacenter/bloop) - Scala build server | [@tues](https://github.com/tues) |
+| [Fury]                | @propensive           |
+| [scala.js] and/or [scalac] | @sjrd |
+| [Hamsters] A mini Scala utility library   | @dgouyette   |
+| IntelliJ Scala plugin | @jastice  |
+| [scalajs-bundler], [endpoints], [scalac] | @ulienrf | 
+| [fs2-cassandra], [fs2-rabbit] and / or [fs2-redis] | @gvolpe |
+| [ZIO] — A principled, powerful, standalone effect data type for any Scala project. | @jdegoes | 
 | Add your project here!                                                                    |                     |
 
+[Fury]:  https://github.com/propensive/fury
+[scala.js]: https://github.com/scala-js/scala-js
+[scalac]: https://github.com/scala/scala
+[Hamsters]: https://github.com/scala-hamsters/hamsters
+[scalajs-bundler]: https://github.com/scalacenter/scalajs-bundler
 [endpoints]: https://github.com/julienrf/endpoints
-[endpoints-issues]: https://github.com/julienrf/endpoints/issues?q=is%3Aissue+is%3Aopen+label%3Aspree
-[krever]: https://github.com/Krever
+[fs2-cassandra]: https://github.com/Spinoco/fs2-cassandra
+[fs2-rabbit]: https://github.com/gvolpe/fs2-rabbit
+[fs2-redis]: https://github.com/gvolpe/fs2-redis
+[ZIO]: https://github.com/scalaz/scalaz-zio
 
 Who will be leading the Scala Open Source Spree?
```

Comment:
I think that this needs to be updated too? @darjutak

Reply:
I think it's correct